### ISSUE TITLE
fails on second run with Centos6.4

### ISSUE THREAD

**Issue by jstrunk** [nvalentine-puppetlabs/puppet-gitolite#1]
I have added the following to the manifest of the system I want to be a gitolite server:

```
    class { "::gitolite":
        path => "/home/git",
        admin_pub_key => "ssh-rsa apubkeyblah jeff@localhost"
    }
```

The first run sets up gitolite successfully. Subsequent runs give the following errors:

```
Error: Could not set 'file' on ensure: No such file or directory - /var/lib/gitolite/admin.pub.puppettmp_2174 at 95:/home/jstrunk/Projects/puppet/modules/gitolite/manifests/package.pp
Error: Could not set 'file' on ensure: No such file or directory - /var/lib/gitolite/admin.pub.puppettmp_2174 at 95:/home/jstrunk/Projects/puppet/modules/gitolite/manifests/package.pp
Wrapped exception:
No such file or directory - /var/lib/gitolite/admin.pub.puppettmp_2174
Error: /File[/var/lib/gitolite/admin.pub]/ensure: change from absent to file failed: Could not set 'file' on ensure: No such file or directory - /var/lib/gitolite/admin.pub.puppettmp_2174 at 95:/home/jstrunk/Projects/puppet/modules/gitolite/manifests/package.pp
Notice: /Stage[main]/Gitolite::Package/Exec[configure gitolite admin]: Dependency File[/var/lib/gitolite/admin.pub] has failures: true
Warning: /Stage[main]/Gitolite::Package/Exec[configure gitolite admin]: Skipping because of failed dependencies
Notice: /Stage[main]/Gitolite::Package/Exec[rename gitolite group to git]: Dependency File[/var/lib/gitolite/admin.pub] has failures: true
Warning: /Stage[main]/Gitolite::Package/Exec[rename gitolite group to git]: Skipping because of failed dependencies
Notice: /Stage[main]/Gitolite::Package/Exec[rename gitolite user to git]: Dependency File[/var/lib/gitolite/admin.pub] has failures: true
Warning: /Stage[main]/Gitolite::Package/Exec[rename gitolite user to git]: Skipping because of failed dependencies
Notice: Finished catalog run in 3.72 seconds
```

**Comment by jstrunk** [comment]
This issue is caused when the gitolite module renames the gitolite user and moves its home directory.

A simple fix is to change the admin.pub file to something in /tmp a la:

```
diff --git a/modules/gitolite/manifests/package/params.pp b/modules/gitolite/man
index f29272c..3bbf036 100644
--- a/modules/gitolite/manifests/package/params.pp
+++ b/modules/gitolite/manifests/package/params.pp
@@ -10,7 +10,7 @@ class gitolite::package::params {
        fail("Sorry, this class supports RedHat version 6.4 only!")
       }
       $epel_url = 'http://mirrors.servercentral.net/fedora/epel/6/i386/epel-rel
-      $admin_pub_key_file = "/var/lib/gitolite/admin.pub"
+      $admin_pub_key_file = "/tmp/admin.pub"
     }

     default: {fail("OS family ${::osfamily} not supported by this class!")}
```

**Comment by nvalentine-puppetlabs** [comment]
Thanks for the bug report and the patch. Somehow I missed the email notification so I'm just getting around to taking a look. Please stay tuned.

**Comment by nvalentine-puppetlabs** [comment]
Thanks for the bug report. This issue is fixed in tag 0.7.0.
